src/coredump/coredump.c

```diff
@@ -1056,6 +1063,8 @@ static int process_socket(int fd) {
         log_debug("Processing coredump received on stdin...");
 
         for (;;) {
+                struct cmsghdr *cmsg;
```

Comment:
please move to innermost scope

Reply:
I am getting rid of this variable completely.